lib/api/api-fetch/headers.js

```diff
@@ -233,8 +245,31 @@ class Headers {
       callback.call(thisArg, this[kHeadersList][index + 1], this[kHeadersList][index], this)
     }
   }
+
+  [Symbol.for('nodejs.util.inspect.custom')] () {
+    return Object.fromEntries(this.entries())
+  }
```

Comment:
This is slightly different than node-fetch. What do we prefer?

Reply:
@Ethan-Arrowood

Reply:
Yeah I missed this when I ported over my Node.js PR changes. In Node core we were just going to return the underlying `this[kHeadersList]`.

Reply:
https://github.com/nodejs/node/pull/38986/files?file-filters%5B%5D=.js&file-filters%5B%5D=.mjs#diff-3569a54af4e25669eee4cc77bd0f4e86ca5fafb0cf766759a4bd2fb482fa0984R266-R268